packages-next/fields/src/types/json/index.ts

```diff
@@ -0,0 +1,22 @@
+import type { FieldType, BaseGeneratedListTypes, FieldDefaultValue } from '@keystone-next/types';
+import { resolveView } from '../../resolve-view';
+import type { FieldConfig } from '../../interfaces';
+import { Json, PrismaJsonInterface } from './Implementation';
+
+export type JsonFieldConfig<TGeneratedListTypes extends BaseGeneratedListTypes> =
+  FieldConfig<TGeneratedListTypes> & {
+    defaultValue?: FieldDefaultValue<string>;
```

Comment:
This should be any json value, right? (I think we have a type for that in @keystone-next/types)